packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -138,6 +145,7 @@ export function makeRumPublicApi(
     }
 
     isAlreadyInitialized = true
+    return { configuration }
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏init should keep returning undefined